packages/generators/utils/entry.ts

```diff
@@ -87,16 +89,17 @@ export default function entry(self: IEntry, answer: {
 	} else {
 		result = self
 			.prompt([
-				InputValidate(
+				Input(
```

Comment:
Have you tested on empty strings when removing this?

Reply:
The InputValidate was used as an Input since we didn't provide it a validator function.
I've added default values to both so now when the answer is an empty string it defaults the entry to "src/index"